mmdet3d/datasets/nuscenes_dataset.py

```diff
@@ -88,26 +88,44 @@ def __init__(self,
             test_mode=test_mode,
             **kwargs)
 
-    def _filter_with_mask(self, ann_info: dict) -> dict:
-        """Remove annotations that do not need to be cared.
-
-        Args:
-            ann_info (dict): Dict of annotation infos.
+    def filter_data(self) -> List[dict]:
```

Comment:
For a better global understanding for users,  we could exlpain all the possible `keys` in filter_cfg in dataset docs or docstring.